tox.ini

```diff
@@ -15,13 +15,14 @@ DJANGO_SETTINGS_MODULE = apptemplates.test.settings
 [tox]
 envlist =
     flake8-py{26,27,32,33,34,35}
-    pylint-py{27,35}-django110
+    pylint-py{27,35}-django{110,111}
     py{26,27}-django14
     py{26,27,32,33}-django{15,16}
     py{27,32,33,34}-django17
     py{27,32,33,34,35}-django18
     py{27,34,35}-django19
     py{27,34,35}-django110
+    py{27,34,35}-django111
```

Comment:
[Django 1.11](https://pypi.python.org/pypi/Django/1.11.3) also supports Python 3.6, let's add this too.

While you're at it, can you also use variable expansion for the Django 1.9 and 1.10 declaration? e.g.
``` bash
py{27,34,35}-django{19,110}
py{27,34,35,36}-django111
```